test/spec/token.js

```diff
@@ -0,0 +1,40 @@
+define(function(require) {
+  var OktaAuth = require('OktaAuth');
```

Comment:
They aren't yet. The core source is in commonjs, but the tests still use this amd style.